Formula/wxwidgets.rb

```diff
@@ -1,4 +1,4 @@
-class Wxmac < Formula
+class Wxwidgets < Formula
   desc "Cross-platform C++ GUI toolkit (wxWidgets for macOS)"
```

Comment:
```suggestion
  desc "Cross-platform C++ GUI toolkit"
```
Could probably just fix this in a syntax-only PR though.

Reply:
Fixed and rebased.